index.html

```diff
@@ -132,16 +132,76 @@ <h2>Abstract</h2>
           </a>
         </li>
       </ol>
+    </div>
+  </section>
 
-      <p id="affiliation">
-        <a href="https://umich.edu/">
-          <img id="um-vertical" alt="University of Michigan" src="img/um-vertical.png">
-        </a>
-      </p>
-
+  <section>
+    <div class="content">
+      <h2>Example</h2>
```

Comment:
What about the following?

```suggestion
      <h2>Example from our Dataset</h2>
```

To give more context to someone who enters our website and doesn't know what it's about.

Reply:
That makes sense to me.